tracking.py

```diff
@@ -840,21 +840,21 @@ def compute_matches(tracks, t, max_age,
       matches[next_inds[i]] = track_idx[prev_inds[i]]
   return matches
 
-def update_tracks(tracks, matches, t, all_boxes, all_keypoints):
-  """Updates the tracks for frame t given the matches, and creates new tracks when necessary.
+def update_tracks(tracks, matches, i, all_boxes, all_keypoints):
+  """Updates the tracks for frame i given the matches, and creates new tracks when necessary.
```

Comment:
abrichr 9 days ago Member
Please add a label to provide context to the reader, e.g.

printf('matches:', matches)